src/pretix/base/exporters/orderlist.py

```diff
@@ -140,8 +142,10 @@ def iterate_orders(self, form_data: dict):
         }
 
         for order in qs.order_by('datetime'):
+            tz = pytz.timezone(order.event.settings.timezone)
+
             row = [
-                order.code,
+                order.full_code if self.is_multievent else order.code,
```

Comment:
I think this difference will add confusion and we should put the event slug in it's own column. Also, order codes should be unique within one organizer, right?

Same for the other sheets